docs/user/orion.rst

```diff
@@ -0,0 +1,92 @@
+.. _orion:
+
+Orion
+=====
+
+.. contents:: Content of this document
+    :local:
+    :depth: 2
+
+Overview
+--------
+
+The `Orion plugin <https://github.com/ls1intum/Orion>`_ for `IntelliJ <https://www.jetbrains.com/idea/>`_ supports students, teaching assistants (tutors), and instructors with the conduction of programming exercises. It integrates Artemis in IntelliJ and automates the download and upload of exercises.
+
+Installation
+^^^^^^^^^^^^
+
+Orion is installed like every other plugin via IntelliJ's marketplace. Alternatively, the latest build can be found in `Orion's GitHub repository <https://github.com/ls1intum/Orion>`_ as described there.
+
+General Usage
+^^^^^^^^^^^^^
+
+After installation, Orion provides a custom tool window at *View -> Tool Windows -> Artemis*. This window contains an integrated browser displaying Artemis.
+
+At the top of the tool window is a back button |back-button|. Clicking it will return to the initial page, that is:
+
+- The exercise details page if an exercise is opened as a student
+- The exercise edit in editor page if an exercise is opened as an instructor
+- The assessment dashboard if an exercise is opened as a tutor, or, if a submission is downloaded, the assessment details page of that submission
+- The Artemis home page if no Artemis project is opened
+
+Settings
+^^^^^^^^
+
+Orion's settings are at *Settings -> Tools -> Orion*. The settings include:
+
+- The Artemis base url. Can be changed to switch to a specific Artemis instance. Defaults to https://artemis.ase.in.tum.de. **Important:** The url must not end with a /, otherwise it will not work!
+- The Artemis exercise paths. Orion will suggest to store newly downloaded exercises at *default path/course/exercise name*, with the default path dependant of the setting.
+- The default commit message
+- An option to change the user agent if needed as well as a button to restart the integrated browser. If Orion gets stuck restarting the browser might solve the issue without restarting IntelliJ.
+
+Problems
+^^^^^^^^
+
+Please report any issues on `GitHub <https://github.com/ls1intum/Orion>`_.
```

Comment:
I would not write only "GitHub" since I assume that most people access the documentation through GitHub anyway. Just another repository. So that's where the focus should lie on.

Reply:
:+1: